### DISCUSSION TITLE
在ROCm 6.22上，使用AMD GPU推理Qwen 2.5 1.5B Insturct时，报错libcudart.so.12 no such file or dierctory

### DISCUSSION THREAD

**Discussion by MicahYM** [QwenLM/Qwen3#1087] [Q&A]
我在安装了ROCm 6.22 的电脑上，使用AMD GPU推理Qwen 2.5 1.5B Insturct时，报错libcudart.so.12 no such file or dierctory
不知道这个问题有人遇到过吗，怎么解决？

**Comment by jklj077** [comment]
AMD卡需要HIP/ROCm版的包，是不是装了CUDA版的。

- pytorch: https://pytorch.org/get-started/locally/
  
  ![image](https://github.com/user-attachments/assets/dfc5eba1-939c-43dd-a142-a3e7afde8517)

- vllm: https://docs.vllm.ai/en/latest/getting_started/amd-installation.html
